File: StrykerDG.FarmForge.Client/farmforge_client/lib/models/dto/split_inventory_dto.dart
class SplitInventoryDTO {
  List<int> productIds;
  int unitTypeConversionId;
  int locationId;

  SplitInventoryDTO({
    this.productIds,
    this.unitTypeConversionId,
    this.locationId
  });

  Map<String, dynamic> toMap() {
    return {
      'ProductIds': this.productIds,
      'UnitTypeConversionId': this.unitTypeConversionId,
      'LocationId': this.locationId
    };
  }
}

File: StrykerDG.FarmForge.Api/Controllers/DTO/Requests/SplitInventoryDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace StrykerDG.FarmForge.LocalApi.Controllers.DTO.Requests
{
    public class SplitInventoryDTO
    {
        public List<int> ProductIds { get; set; }
        public int UnitTypeConversionId { get; set; }
        public int LocationId { get; set; }
    }
}
